### ISSUE TITLE
rnnt complie error on TensorRT

### ISSUE THREAD

**Issue by Bob123Yang** [mlcommons/ck#1021]
Do full accuracy and performance runs for all the scenarios with below commands:

**cmr "generate-run-cmds inference _submission _all-scenarios" --model=rnnt \
--device=cuda --implementation=nvidia-original --backend=tensorrt \
--execution-mode=valid --results_dir=$HOME/results_dir \
--category=edge --division=open --quiet**

**Comment by arjunsuresh** [comment]
yes, the `submission` variation turns on all the modes as given [here](https://github.com/mlcommons/ck/blob/master/cm-mlops/script/run-mlperf-inference-app/_cm.json#L407)

@Bob123Yang Can you please share the compilation error mentioned in the title? The compilation of nvidia-harness is not tested on a wide variety of systems and it is recommended to us [this docker setup](https://github.com/mlcommons/ck/blob/master/cm-mlops/script/reproduce-mlperf-inference-nvidia/README-about.md#using-docker-recommended-on-x86-systems) for running nvidia-harness.

**Comment by gfursin** [comment]
Hi @Bob123Yang . Did this command work for you? Should we close this ticket? Thanks!
